### PR TITLE
feat: Refactor FX APIs under dynamo namespace for parity with TS APIs

### PR DESCRIPTION
**Pull request by peri044**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

**Comment by narendasan** [comment]
@peri044 only copy over the files you made changes to to the new directory. You can just import the rest from .fx

**Comment by frank-wei** [comment]
This PR is quite bulky...  Please add or move only necessary ones. Consider code reuse could help to reduce the change size.

**Review by narendasan — COMMENTED** [review]
Remove the converter tests, these do not need to be duplicated unless they are changed

**Comment by narendasan** [comment]
@gs-olive is there anything blocking this merge? 
When merged can you cherry pick into 1.4?

File: py/torch_tensorrt/_Device.py [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-from torch_tensorrt import _C
-
 import warnings
 
+try:
+    from torch_tensorrt import _C
+except:
+    warnings.warn(
+        "Unable to import _C extension of Torch-TensorRT. Some methods might be unavailable. You can ignore this error if you're \
+                  not using any functions dependent on internal C++ APIs"
```
Comment (narendasan) [inline py/torch_tensorrt/_Device.py:13]: Users will not know what this means. Probably rename this to something like 

```
Unable to import torchscript frontend core and torch-tensorrt runtime. Some dependent features may be unavailable.
